src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java

```diff
@@ -64,4 +64,17 @@ public Job launchTestJob(
           });
     }
 
+    @ApiOperation(value = "Launch Job to Milk the Cows")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/launch/milkjob")
```

Comment:
This should be a @PostMapping not a @GetMapping

The reason is that GET requests are not supposed to update anything.  But this job will definitely update state on the server.  Therefore, it is supposed to be a POST.

Reply:
Fixed!